app/src/main/AndroidManifest.xml

```diff
@@ -13,12 +13,12 @@
       android:theme="@style/AppTheme">
     <activity
         android:name=".MainActivity"
-        android:screenOrientation="landscape">
+        android:screenOrientation="sensor">
```

Comment:
Question: What’s the intention behind this?

Reply:
Use the trivial drive application in Portrait.